boards/arm/mec172xevb_assy6906/mec172xevb_assy6906_defconfig

```diff
@@ -18,5 +18,6 @@ CONFIG_UART_CONSOLE=y
 CONFIG_ADC=y
 CONFIG_I2C=y
 CONFIG_ESPI=y
+CONFIG_PS2=y
```

Comment:
This whole file needs to be cleaned up to include only the drivers needed for hello_world

Reply:
okay, will remove adc, i2c, espi, ps2, spi, spi_async

Reply:
including only drivers for hello_world